src/features/klant/service.ts

```diff
@@ -291,17 +360,115 @@ export const useKlantByVestigingsnummer = (
   });
 };
 
+export type CreateBedrijfKlantIdentifier =
```

Comment:
Overlap met BedrijfSearchParameter. Kunnen we niet een gedeeld BedrijfIdentifier type maken? kvknummer is hier zonder hoofdletter N. Goed om daar consequent in te zijn denk ik?

Reply:
nee, volgens mij komt dat niet uit. ze hebben een ander doel en andere waardes. de naamgeving kan misschien duidelijker